### PR TITLE
 NOTIF-365 admin ui bundle name issue

### PR DESCRIPTION
**Pull request by aferd**
https://user-images.githubusercontent.com/86322239/141199186-93bff849-d3c1-4cc1-b519-33d5d6df9bc3.mov

### CODE REVIEW

**Comment by josejulio** [comment]
I think some unreleated code got mixed in this PR.
Anyhow, my suggestion would be to use that `id` to load the bundle (while it's loading it could show a skeleton in there). See the breadcrumb interaction when clicking a policy: https://console.redhat.com/insights/policies/list

File: admin-console/src/main/webapp/package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "react-scripts": "4.0.3",
     "react-use": "17.3.1",
     "typescript": "4.4.3",
+    "typestyle": "^2.1.0",
```
Comment (josejulio) [inline admin-console/src/main/webapp/package.json:26]: Lets pin the dependency. (will need a `yarn install` to refresh the `yarn.lock`) 

```suggestion
    "typestyle": "2.1.0",
```

File: admin-console/src/main/webapp/src/pages/ApplicationPage.tsx [hunk 2]
```diff
@@ -1,58 +1,78 @@
...
     const applicationTypesQuery = useApplicationTypes(applicationId);
-    const getBundle = useBundles();
     const columns = [ 'Event Type', 'Name', 'Description', 'Event Type Id' ];
 
     const newEvent = useCreateEventType();
-    const [ id ] = React.useState<string | undefined>();
-    const [ displayName, setDisplayName ] = React.useState<string | undefined>();
-    const [ name, setName ] = React.useState<string | undefined>();
-    const [ description, setDescription ] = React.useState<string | undefined>();
+    const [ event, setEvent ] = React.useState<Partial<EventType>>({});
```
Comment (josejulio) [inline admin-console/src/main/webapp/src/pages/ApplicationPage.tsx:38]: Is this unrelated to this PR? Maybe waiting on the other PR or reverting 74d2ae037c2c42efb5d2a81b11cbf7e574ec5963 ?

**Comment by aferd** [comment]
https://user-images.githubusercontent.com/86322239/142267058-5a5dce3c-5fe3-418e-bae0-c04d80b6b9d6.mov

**Review by josejulio — APPROVED** [review]

**Pull request merged**